spec/services/figures_extractor_spec.rb

```diff
@@ -0,0 +1,17 @@
+require "rails_helper"
+
+describe FiguresExtractor do
+  let(:image_stream) { FileStringIO.new("sample-image.gif", File.open(Rails.root.join('spec', 'fixtures', '5x5-sample-image.gif'), 'rb').read) }
```

Comment:
Line is too long. [144/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.